README.md

```diff
@@ -456,7 +456,9 @@ ANSWERS.addComponent('SearchBar', {
     enabled: false,
     // Optional, the message in the alert. Defaults to the below
     message: "We are unable to determine your location"
-  }
+  },
+  // Optional, a callback invoked when the clear search button is clicked
+  onClearSearch: function() {}
```

Comment:
This component has many config options and they're all on the top-level. There's no config hierarchy, save for the `geolocation` stuff. What would you think about adding a `customHooks` config option (or something like that) that's an `Object`? The `Object` would have an `onClearSearch` entry, `onSubmit`, etc. I really think that grouping like config options together helps with progressive disclosure of complexity and compartmentalization.

Reply:
Yes totally agree! Updated to have a `customHooks` option.